lib/phoenix/transports/v2/longpoll_serializer.ex

```diff
@@ -12,7 +12,7 @@ defmodule Phoenix.Transports.V2.LongPollSerializer do
     encoded =
       %Message{topic: msg.topic, event: msg.event, payload: msg.payload}
       |> to_list()
-      |> Poison.encode!()
+      |> Jason.encode!()
```

Comment:
We might use `encode_to_iodata!` here as well, like in the websocket serializer.